src/xenialdan/Backpack/BackpackMenuMetadata.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace xenialdan\Backpack;
+
+use muqsit\invmenu\inventory\InvMenuInventory;
+use muqsit\invmenu\metadata\SingleBlockMenuMetadata;
+
+class BackpackMenuMetadata extends SingleBlockMenuMetadata
```

Comment:
lol wth is this

Reply:
New API method to specify inventory, MenuMetadata holds the graphical stuff that needs to be sent